src/main/java/org/opensearch/security/user/ServiceAccountRegistrationResponse.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2015-2018 _floragunn_ GmbH
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.security.user;
+
+
+public class ServiceAccountRegistrationResponse {
```

Comment:
This class doesn't seem to be used?

Reply:
Yeah, this is for once the core side is hooked up. It will be what is sent back to core. I can remove it for now if you want? What do you think?

Reply:
I would prefer we removed all unused code / comments.  Is there a meta issue with this step called out that we could reference in future PRs?

Reply:
Alright I will remove it; it is documented as part of the api handling so should be all set